internal/engine/engine.go

```diff
@@ -224,7 +224,7 @@ func (c *CraneEngine) ExecuteChecks(ctx context.Context) error {
 		checkElapsedTime := time.Since(checkStartTime)
 
 		if err != nil {
-			logger.WithValues("result", "ERROR", "err", err).Info("check completed", "check", check.Name())
+			logger.WithValues("result", "ERROR", "err", fmt.Sprintf("%v", err)).Info("check completed", "check", check.Name())
```

Comment:
Should be able to just call the Error() method here.

```suggestion
			logger.WithValues("result", "ERROR", "err", err.Error()).Info("check completed", "check", check.Name())
```

Reply:
Done